### PR TITLE
Implement tags page 

### PR DESCRIPTION
**Pull request by VkoHov**
- Implement Tags page
- Implement `CreateTag` page
- Implement `TagDetail` page
- Implement `TagsForm` component
- Implement tagsService with getTags, getTagById, createTag, updateTag methods

### CODE REVIEW

**Review by roubkar — APPROVED** [review]

File: aim/web/ui_v2/src/components/TagForm/TagForm.tsx [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        {editMode ? (
+          <Button onClick={onSaveButtonClick}>Save</Button>
+        ) : (
+          <Button onClick={onCreateButtonClick}>Create</Button>
+        )}
+        {editMode ? (
+          <Button onClick={onResetButtonClick}>Reset</Button>
+        ) : (
+          <NavLink to='/tags'>Cancel</NavLink>
+        )}
```
Comment (arsengit) [inline aim/web/ui_v2/src/components/TagForm/TagForm.tsx:117]: @VkoHov I think its not needed to check editMode twice

File: aim/web/ui_v2/src/pages/Tags/CreateTag.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React, { memo } from 'react';
+import { Divider, Grid } from '@material-ui/core';
+import TagForm from 'components/TagForm/TagForm';
+
+import './Tags.scss';
+
+function CreateTag(): React.FunctionComponentElement<React.ReactNode> {
+  return (
+    <section className='Tags'>
+      <Grid container justify='center'>
```
Comment (arsengit) [inline aim/web/ui_v2/src/pages/Tags/CreateTag.tsx:10]: justify prop is deprecated, its changed to justifyContent

**Review by arsengit — APPROVED** [review]
lgtm

File: aim/web/ui_v2/src/utils/d3/drawHoverAttributes.ts [hunk 3]
```diff
@@ -34,7 +34,8 @@ function drawHoverAttributes(props: IDrawHoverAttributesProps): void {
     syncHoverState,
   } = props;
 
-  const chartRect: DOMRect = visAreaRef.current?.getBoundingClientRect() || {};
+  const { top: chartTop, left: chartLeft }: { top: number; left: number } =
```
Comment (KaroMourad) [inline aim/web/ui_v2/src/utils/d3/drawHoverAttributes.ts:37]: i think this is not correct to delete this part of code

File: aim/web/ui_v2/src/utils/d3/drawHoverAttributes.ts [hunk 4]
```diff
@@ -335,36 +336,16 @@ function drawHoverAttributes(props: IDrawHoverAttributesProps): void {
     );
   }
 
-  function getBoundedPosition(
```
Comment (KaroMourad) [inline aim/web/ui_v2/src/utils/d3/drawHoverAttributes.ts:338]: and this part

File: aim/web/ui_v2/src/utils/d3/drawHoverAttributes.ts [hunk 5]
```diff
@@ -335,36 +336,16 @@ function drawHoverAttributes(props: IDrawHoverAttributesProps): void {
...
-    const [xMin, xMax] = attributesRef.current.xScale.range();
-
-    return {
-      topPos: yPos > yMax ? yMax : yPos < yMin ? yMin : yPos,
-      leftPos: xPos > xMax ? xMax : xPos < xMin ? xMin : xPos,
-    };
-  }
-
   function getActivePoint(circle: INearestCircle): IActivePoint {
-    const xPos = circle.x;
```
Comment (KaroMourad) [inline aim/web/ui_v2/src/utils/d3/drawHoverAttributes.ts:355]: and this part too

**Review by KaroMourad — COMMENTED** [review]
lgtm

**Pull request merged**